crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -187,6 +187,55 @@ type Epoch {
 	protocolConfigs: ProtocolConfigs
 }
 
+type Event {
+	id: ID!
+	sendingModuleId: MoveModuleId
+	eventType: MoveType
+	senders: [Address!]
```

Comment:
I think the type for this should be Option<Vec<Option<Address>>>, if we want to conform to the desired schema.

Reply:
Yeah having sat on this for a bit, I don't think we'd want [Address], an array of nullable Address, which means something like [Address, null, Address] is possible.

Reply:
ok, I kept it as per the schema so we should revisit this design choice next week.